packages/flutter/lib/src/widgets/text.dart

```diff
@@ -164,8 +164,8 @@ class DefaultTextStyle extends InheritedWidget {
 /// for example, to make the text bold while using the default font family and
 /// size.
 ///
-/// To display text that uses multiple styles (e.g., a paragraph with some bold
-/// words), use [RichText].
+/// The [Text] widget can also be created with a [TextSpan] to display text that
+/// uses multiple styles (e.g., a paragraph with some bold words).
```

Comment:
maybe explicitly say `Using the [new Text.rich] constructor, the [Text] widget can also be created with a [TextSpan] ...`

(`[new Foo]` is how you refer to constructors in dartdocs)